example/html/config.json

```diff
@@ -0,0 +1,75 @@
+{
+  "analytics": true,
+  "animatedCard": true,
+  "deferInit": false,
```

Comment:
there are two properties left: 
-  `"fieldsToSubmit": ["pan", "expirydate", "securitycode"],`
-  ` "cybertonica": {
    "apikey": ""
  },`
I'm not sure about the second property, because we must merge Cybertonica branch first into develop (I'm waiting for changes in js-payments-testing master branch)